lib/embed.dart

```diff
@@ -825,6 +848,45 @@ class Embed extends EditorUi {
     return success;
   }
 
+  void _toggleCodeInput() {
+    var editorAndConsoleContainer =
+        querySelector('#editor-and-console-container');
+    var webOutput = querySelector('#web-output');
+
+    var isEditorHidden = editorAndConsoleContainer.hidden;
+
+    if (isEditorHidden) {
+      // show code input, hide UI output
+      editorCodeInputTabButton.text = 'Hide code';
+      editorAndConsoleContainer.removeAttribute('hidden');
+      webOutput.setAttribute('hidden', '');
+      _updateShowcase(isEditorVisible: true);
+
+      // run format to force to display the code & show caret in the editor
+      _format();
```

Comment:
a "hacky" way to show the code when toggled, without `_format();` it does not display the code immediately, only after clicking into the editor window (plus it does not show caret).

Reply:
seems like that this workaround is not working now, hmm